examples/7guis/crud.slint

```diff
@@ -19,12 +23,27 @@ MainWindow := Window {
             model: root.names-list;
         }
 
-        Text { col: 2; row: 1; text: "Name: "; }
+        Text {
+            col: 2;
+            row: 1;
+            text: "Name: "; 
+            vertical-alignment: center;
+            horizontal-alignment: right;
```

Comment:
These "Labels" aren't aligned to the right in the official screenshots.
But it looks better that way.